Update license.mdx (#7815)
hashicorp/consul: website/pages/docs/commands/license.mdx

        Redundancy Zone
        Advanced Network Federation
```

## reset

> Resets a license in the datacenter to the builtin on if it is still valid. 

Reviewer: ```suggestion
Resets license for the datacenter to the one builtin in Consul binary, if it is still valid. 
```
